lib/apicasso/active_record_extension.rb

```diff
@@ -8,7 +8,9 @@ module Apicasso
   # own application.
   module ActiveRecordExtension
     extend ActiveSupport::Concern
+    # Module with class methods of Apicasso
     module ClassMethods
+      # Method that validate ActiveRecord
```

Comment:
This method validates nothing. It is used to map validations for consumption on the Swagger JSON.